docs/products/runs/auto-resume-experiments.md

```diff
@@ -48,4 +48,7 @@ grid run --use_spot --auto_resume --instance_type p3.2xlarge mnist.py
 
 #### UI
 
-![image](https://user-images.githubusercontent.com/13732925/148102089-f540356a-a2e6-4e9d-ac1f-51de26691086.png)
\ No newline at end of file
+![image](https://user-images.githubusercontent.com/13732925/148102089-f540356a-a2e6-4e9d-ac1f-51de26691086.png)
+
+#### Lightning Fault Tolerance
+Alternatively, you can enable autoresume experiments via Lightnings Fault tolerance feature see [here](https://pytorch-lightning.readthedocs.io/en/latest/advanced/fault_tolerant_training.html#:~:text=Fault%2Dtolerant%20Training%20is%20an,a%20hardware%20or%20software%20failure.&text=fit()%20fails%20in%20the,and%20everything%20will%20be%20restored.) for the offcial lightning documentation or [here](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/fault_tolerant/automatic.py) for a tested example using the fault tolerance feature and Grid. At this time there are some caveats with reloading.
```

Comment:
```suggestion
#### PyTorch Lightning Fault Tolerance

PyTorch Lightning provides a built-in auto-resume mechanism. This would enable you to run your experiments at the cheapest cost without worrying about your experiments being terminated before they should be.  

You can learn more about the PyTorch Lightning Fault Tolerance mechanism [here](https://pytorch-lightning.readthedocs.io/en/latest/advanced/fault_tolerant_training.html#:~:text=Fault%2Dtolerant%20Training%20is%20an,a%20hardware%20or%20software%20failure.&text=fit()%20fails%20in%20the,and%20everything%20will%20be%20restored.).

Furthermore, PyTorch Lightning provides a reproducible script that you can find [here](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/fault_tolerant/automatic.py) which is tested end-to-end on Grid with the Auto-Resume feature and should help you get started.
```